src/deluge/gui/views/automation_view.cpp

```diff
@@ -3055,17 +3055,21 @@ void AutomationView::selectNonGlobalParam(int32_t offset, Clip* clip) {
 // used with SelectEncoderAction to get the next midi CC
 void AutomationView::selectMIDICC(int32_t offset, Clip* clip) {
 	if (isOnAutomationOverview()) {
-		clip->lastSelectedParamID = 0;
```

Comment:
I noticed that when you're on automation overview and scroll right, that it jumps to CC 2 and you have to scroll back to get to CC 0. Can you fix it so the scrolling right when on automation overview brings you to CC 0? Thanks